bin/burden

```diff
@@ -2196,16 +2196,47 @@ verify_test_def_file()
 verify_os_vendor()
 {
 	os_vendor_found=0
+	os_name=$(echo "$1" | awk -F ":" '{ print $1 }')
+	os_args=$(echo "$1" | awk -F ":" '{ print $2 }')
 
-	for os_vendor_check in  $gl_valid_os_vendors; 
+	for os_vendor_check in $gl_valid_os_vendors;
 	do
-		if [[ $1 == $os_vendor_check ]]; then
+		if [[ $os_name == $os_vendor_check ]]; then
 			os_vendor_found=1
+			gl_os_vendor=$os_name
 		fi
 	done
 	if [[ $os_vendor_found -eq 0 ]]; then
 		cleanup_and_exit "Error: $1, is an unrecognized OS vendor.  Possible values are: $gl_valid_os_vendors" 1
 	fi
+
+	if [ -n "$os_args" ]; then
+		echo $os_args
+		process_os_args "$os_args"
+	fi
+}
+
+process_os_args() {
+	arg_file=$(mktemp)
```

Comment:
So if we see the tmp file for some reason we know what it is.
mktemp /tmp/zath_prcs_os_args.XXXXX